Dockerfile.windows

```
@@ -155,7 +160,11 @@ FROM deps AS build
 ARG HOUNDTTS_VERSION=0.0.0
 
 WORKDIR C:\\src
-COPY . C:\\src\\
+COPY CMakeLists.txt C:\\src\\
```

Hadolint (2.14.0)

[error] unexpected newline
expecting '\', --checksum, --chmod, --chown, --exclude, --link, a space separated list of file paths, at least one space, only one --from, or other flag

(DL1000)